lib/std/dwarf.zig

```diff
@@ -522,6 +527,10 @@ fn parseFormValue(allocator: mem.Allocator, in_stream: anytype, form_id: u64, en
 
         FORM.string => FormValue{ .String = try in_stream.readUntilDelimiterAlloc(allocator, 0, math.maxInt(usize)) },
         FORM.strp => FormValue{ .StrPtr = try readAddress(in_stream, endian, is_64) },
+        FORM.strx1 => return FormValue{ .StrOffset = @intCast(u64, try in_stream.readInt(u8, endian)) },
```

Comment:
If it's not too much trouble, it'd be really nice to add branches for the other missing DWARFv5 forms: `strx`, `addrx`, `addrx1`, `addrx2`, `addrx3`, `addrx4`, `data16`, `implicit_const`, `line_strp`, `loclistx`, `rnglistx`, `ref_sup4`, `ref_sup8`, `strp_sup`

You don't have to implement these now, but adding a good error or panic would be useful.

Reply:
Yes, I think this is a fair point. 

How would you like the error to be reported? There are two ways I could do it.

1. Just return a normal error E.g `error.FormNotImplementedAddrx`. This will always result in broken stack trace, even if the data in question is never actually requested.
2. Consume the number of bytes that correspond to the form type and log an error / warning.

Reply:
I'd say just add `@panic("Could not print stack trace: DWARFv5 not fully supported")`. Normally we'd use `unreachable`, but we can only track that one down if backtraces are working so that won't do 😛

Returning a normal error is also fine by me, but the error handling for stack-trace printing will need to be updated, too.

BTW there's one more item other than these forms to round out DWARFv5 support: the `.debug_line` section has a V5 format that we can't parse yet